av1an-cli/src/lib.rs

```diff
@@ -167,5 +171,5 @@ pub fn parse_args() -> String {
 
 /// Get default values of args
 pub fn default_args() -> String {
-  serde_json::to_string(&Args::parse_from(["av1an"])).unwrap()
+  serde_json::to_string(&Args::parse_from(&["av1an"])).unwrap()
```

Comment:
this should be fine as it is you used nightly rust tool-chain (as recommended for pyo3)

Reply:
Should i return it to how it was then?

Reply:
I think it's fine